internal/provider/generators/schema/main.go

```diff
@@ -449,6 +457,7 @@ func (g *Generator) GenerateResources(packageName, filename, generatedCodeRootDi
 		ImportPathSuffixes:             importPathSuffixes,
 		PackageName:                    packageName,
 		Resources:                      resources,
+		TemplatesRoot:                  templatesRoot + "/resources",
```

Comment:
This puts all the metadata files in the same folder as the template files. This is a bit unwieldy as there will be a metadata file per resource. Considering moving metadata files under a `/metadata` subfolder under `templates/resources`.

Reply:
Let's put the files in the new subfolder if that's OK.